addons/editor-number-arrow-keys/userscript.js

```diff
@@ -94,12 +94,21 @@ export default async function ({ addon }) {
 
     e.preventDefault();
 
-    const changeBy =
-      (e.shiftKey
+    let changeBy = e.code === "ArrowUp" ? 1 : -1;
+    if (addon.settings.get("useCustom")) {
+      // TODO: add proper float settings
+      changeBy *= e.shiftKey
+        ? parseFloat(addon.settings.get("shiftCustom"))
+        : e.altKey
+        ? parseFloat(addon.settings.get("altCustom"))
+        : parseFloat(addon.settings.get("regularCustom"));
```

Comment:
Yeah, we need to be careful with NaN here. We should early-return or change by zero if the string setting can't be converted into a number.